code/__HELPERS/qdel.dm

```diff
@@ -1,5 +1,5 @@
-#define QDEL_IN(item, time) addtimer(CALLBACK(GLOBAL_PROC, .proc/qdel, item), time, TIMER_STOPPABLE)
-#define QDEL_IN_CLIENT_TIME(item, time) addtimer(CALLBACK(GLOBAL_PROC, .proc/qdel, item), time, TIMER_STOPPABLE | TIMER_CLIENT_TIME)
+#define QDEL_IN(I,T) addtimer(CALLBACK(I, /datum/proc/qdel_self), T, TIMER_STOPPABLE)
```

Comment:
why give the arguments less readable names

Reply:
Addressed!